hawkular-alerts-engine/src/main/resources/org/hawkular/alerts/schema/updates/schema-1.2.1.groovy

```diff
@@ -29,4 +29,5 @@ CREATE TABLE sys_config (
         PRIMARY KEY (config_id, name)
 )
 """
+  verify { tableExists(keyspace, "sys_config") }
```

Comment:
This is very nice @jsanda.
Is there a similar way for check columns ? thinking on alter table sentences.
It is so, I will change a legacy checker inside alerting.
No blocker, this PR can go as it is, these enhancements can go in a following PR and not needed for this version.